tests/test_review_comments.py

```
@@ -0,0 +1,557 @@
+
+    def test_fenced_json(self) -> None:
+        text = '```json\n{"findings": [], "summary": "clean"}\n```'
+        findings, summary = _parse_findings(text)
+        assert findings == []
+        assert summary == "clean"
+
+    def test_fenced_no_lang(self) -> None:
+        finding = '{"file": "b.py", "severity": 5, "category": "test", "description": "x"}'
+        inner = f'{{"findings": [{finding}], "summary": "s"}}'
+        text = f"```\n{inner}\n```"
+        findings, summary = _parse_findings(text)
```

Ruff (0.16.1)

[warning] Unpacked variable `summary` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)